pb/rpc.proto

```diff
@@ -45,6 +45,12 @@ message ControlGraft {
 
 message ControlPrune {
 	optional string topicID = 1;
+	repeated PeerInfo peers = 2;
+}
+
+message PeerInfo {
+	optional bytes peerID = 1;
+	optional bytes signedAddrs = 2;
```

Comment:
Nit: really strange that we're using `bytes` instead of a predefined format we already know (a signed envelope of `RoutingStateRecord`s). Is it just too difficult with our modular libraries to do this properly?

Reply:
There is a reason for this: we don't want to care about the details of the routing state envelope or have to unmarshal it unnecessarily.
Also, we don't want to have a dependency on another protobuf (with import), as this inhibits other languages.

Doing it as bytes _is_ the proper way.